src/v2/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -8,47 +8,62 @@ import * as Schema from "v2/Artsy/Analytics/Schema"
 import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "v2/Components/FollowButton/FollowArtistButton"
 import React, { Component } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
-import styled from "styled-components"
 import { get } from "v2/Utils/get"
 import { openAuthToFollowSave } from "v2/Utils/openAuthModal"
 import { Media } from "v2/Utils/Responsive"
-import { ArtistIndicator } from "./ArtistIndicator"
 import { highestCategory } from "./MarketInsights/MarketInsights"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
 
-/**
- * This H1 and H2 were added for SEO purposes
- * TODO: Remove when palette provides the ability to override typography element
- */
-const H1 = styled.h1`
-  all: initial;
-  all: unset;
-  margin: 0;
-  padding: 0;
-  font: normal;
-  font-family: inherit;
-  font-size: medium;
-  font-style: normal;
-  font-variant: normal;
-  font-weight: normal;
-  letter-spacing: normal;
-  line-height: normal;
-`
+const CATEGORIES = {
+  "blue-chip": "Blue Chip Representation",
+  "top-established": "Established Representation",
+  "top-emerging": "Emerging Representation",
+}
+
+const getHighCategory = (artist: ArtistHeader_artist) => {
+  const {
+    artistHighlights: { partnersConnection },
+  } = artist
+  return highestCategory(partnersConnection?.edges)
+}
+
+const getTopAuctionResult = (artist: ArtistHeader_artist) => {
+  return get(
+    artist,
+    () => artist.auctionResultsConnection.edges[0].node.price_realized.display
+  )
+}
+
+const formatFollowerCount = (n: number) => {
+  try {
+    const formatter = Intl.NumberFormat("en-US", {
+      // https://github.com/microsoft/TypeScript/issues/36533
+      // @ts-ignore
+      notation: "compact",
+      compactDisplay: "short",
+    })
+
+    return formatter.format(n).toLocaleLowerCase()
```

Comment:
Is likely fine but https://caniuse.com/mdn-javascript_builtins_intl_numberformat